dnf/cli/commands/repoquery.py

```diff
@@ -435,7 +435,6 @@ def _add_add_remote_packages(self):
 
     def run(self):
         if self.opts.querytags:
-            print(_('Available query-tags: use --queryformat ".. %{tag} .."'))
```

Comment:
I'm fine personally with removing this, but I'd like a second opinion just to make sure someone doesn't find it actually userful, @dmach @j-mracek @m-blaha, do you agree with removing this line?

Reply:
Also if you wanna do this, could you also remove at least the leading line-break (and ideally the trailing one too) from the `QUERY_TAGS` variable? Now it prints an empty line before and after the list of tags which are superfluous.

Reply:
Removed spaces. Although I would also split `reponame (repoid)` into separate items, or removed `reponame` completely. Then it will be relatively easy to parse the output with something like `bash`.

Reply:
It appears `reponame` doesn't stand for the name, it's the id, same as `repoid`. An obvious error? I'm not against just removing it, but I'm not sure we can do that. I'm sure @j-mracek will know more though?

Reply:
I am sorry, but I do not understanding your question. Reponame as an attribute of package class. --qf is capable to return any attribute of Package class.
```
  .. attribute:: reponame

    Id of repository the package was installed from (string).
```

Reply:
> I am sorry, but I do not understanding your question. Reponame as an attribute of package class. --qf is capable to return any attribute of Package class.

The question is whether we should remove `reponame` from the list, because it contains the ID, not the name. And since there already is `repoid`, it is redundant in addition to wrong.

> ```
>   .. attribute:: reponame
> 
>     Id of repository the package was installed from (string).
> ```

I'd assume this is some sort of legacy API attribute, since it's called "name" and contains the ID. FTR the excerpt is from the API documentation: https://github.com/rpm-software-management/dnf/blob/master/doc/api_package.rst. Interestingly `repoid` isn't listed there and also no way to get the actual repo name.

Reply:
I don't see any benefit in removal of `print(_('Available query-tags: use --queryformat ".. %{tag} .."'))`

Reply:
@j-mracek it is to make the output suitable for scripting. For example https://github.com/rpm-software-management/libdnf/issues/720#issuecomment-631625069

Reply:
Although I'm not entirely sure what would you parse the list for (ultimately you have to know the semantics of each field so you need to have them listed somewhere in your code anyway), I find it reasonable to remove it...